pd3/src/main/resources/templates/public/register.html

```diff
@@ -13,7 +13,7 @@
 		<form action="#" th:action="@{/public/register}" th:object="${registerUserModel}" method="post" class="default">
 			<label>E-Mail-Adresse:</label><input type="text" th:field="*{email}" /> <label th:if="${#fields.hasErrors('email')}" th:errors="*{email}"></label><br/>
 			<label>Passwort:</label><input type="password" th:field="*{password}" /> <label th:if="${#fields.hasErrors('password')}" th:errors="*{password}"></label><br/>
-			<label>Passwort wiederholen:</label><input type="password" th:field="*{passwordRepeat}" /> <label th:if="${#fields.hasErrors('passwordRepeat')}" th:errors="*{passwordRepeat}"></label><br/>
+			<label>Passwort wiederholen:</label><input type="password" th:field="*{passwordRepeat}" /> <label th:if="${#fields.hasErrors('passwordRepeat')}" th:errors="*{passwordRepeat}"></label><label th:if="${#fields.hasErrors('global')}" th:errors="*{global}"></label><br/>

```

Comment:
Ja, siehst du richtig und ist auch richtig unschön. Wäre mir eine Freude da mit dir nochmal auf die Suche nach einer besseren Lösung zu gehen. Ggf. validieren wir sowas auch ganz anders (jscript o.ä.).
Es funzt so erstmal, da es keine anderen globalen Fehler gibt. Aber spätestens wenn man email and emailRepeat hat, wirds wieder komisch.